gammapy/scripts/image_pipe.py

```diff
@@ -259,9 +272,11 @@ def __init__(self, empty_image=None,
         if exclusion_mask:
             self.maps['exclusion'] = exclusion_mask
         self.ncounts_min = ncounts_min
+        self.psfmeantab = None
+        self.thetapsf = None
 
-    def make_images(self, make_background_image=False, bkg_norm=True, spectral_index=2.3, for_integral_flux=False,
-                    radius=10):
+    def make_images(self, make_background_image=False, bkg_norm=True, spectral_index=2.3,
+                    radius=10, make_psf=False, region_center=None):
```

Comment:
I think the `make_psf` and `region_center` arguments should now be removed from this `make_images` method (and the docstring as well), because they aren't used any more?